src/main/java/com/neuvector/Scanner.java

```diff
@@ -322,6 +326,14 @@ private static String getMountPath(NVScanner nvScanner){
         return mountPath;
     }
 
+    private static String appendBindMountSharedSuffixIfRequired(final Boolean isBindMountShared, final String mountPath) {
+        if(isBindMountShared != null && isBindMountShared){
+            return new StringBuilder(mountPath).append(":z").toString();
+        }
+
+        return mountPath;
```

Comment:
It would be easier to log the mount path generated here. We do not need a new method. In other words, we do not need: `getVolumeArgs`

Reply:
Commit: [95ff98f](https://github.com/neuvector/scanner-java-wrapper/pull/11/commits/95ff98fa748659b82ad5527cf1b202fa6f2df662)

Reply:
Commit: [1319192](https://github.com/neuvector/scanner-java-wrapper/pull/11/commits/131919257ecf32141f73d81fefd5e94d15a19224)